Classes/Controller/TableOfContentsController.php

```diff
@@ -44,7 +44,9 @@ class TableOfContentsController extends AbstractController
     public function __construct()
     {
         // Read plugin TS configuration.
+        // TODO: This is more or less obsolete - the table of document plugin does not use the TypoScript anymore
         $this->pluginConf = $GLOBALS['TSFE']->tmpl->setup['plugin.']['tx_dlf_tableofcontents.'];
+        $this->initialize();
```

Comment:
Since `initialize()` is called in `AbstractController::__construct`, it seems to make sense to just call `parent::__construct()` at the top of this constructor.

Reply:
Good point. I'll do like this.